src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -447,40 +447,50 @@ def _create_zone():
                 )
             )
 
-        def _update_zone(current, new):
+        def _update_zone(new):
             new.spec.gateway.instances = spec.placement.count or 1
             return new
 
         return self._create_or_patch(
             cos.CephObjectStore, 'cephobjectstores', name,
             _update_zone, _create_zone)
 
-    def add_nfsgw(self, spec):
+    def apply_nfsgw(self, spec):
+        # type: (ServiceSpec) -> None
```

Comment:
For the make check failures:

```suggestion
    def apply_nfsgw(self, spec: ServiceSpec):
        # type: (ServiceSpec) -> None
```

(also need to add import for `NFSServiceSpec` type)

Reply:
Done